reviewers.js

```diff
@@ -1,59 +1,70 @@
 
-/**************************************************************
- - Receives a reviewer name (string) and an array of reviewer objects,
-   and returns the reviewer object with the same name as the reviewerName provided. 
- - Bonus: uses iteration method .find()
- ****************************************************************/
-function getReviewerByName(reviewerName, reviewers) {
-  //TODO: ADD YOUR CODE HERE
-}
+// /**************************************************************
+//  - Receives a reviewer name (string) and an array of reviewer objects,
+//    and returns the reviewer object with the same name as the reviewerName provided. 
+//  - Bonus: uses iteration method .find()
+//  ****************************************************************/
+// function getReviewerByName(reviewerName, reviewers) {
+//   return reviewers.find((value)=>{
+//    return value.reviewerName ===reviewerName
+//   })
+// }
 // console.log(getReviewerByName("Michelle Obama", reviewers));
 
-/**************************************************************
-- Receives a review title (string) and an array of reviewer objects, 
-  and returns the reviewer object that has done a review with the review title provided. 
-- Bonus: uses iteration methods .find() and .some()
- ****************************************************************/
+// /**************************************************************
+// - Receives a review title (string) and an array of reviewer objects, 
+//   and returns the reviewer object that has done a review with the review title provided. 
+// - Bonus: uses iteration methods .find() and .some()
+// //  ****************************************************************/
 function getReviewerByReviewTitle(reviewTitle, reviewers) {
-  //TODO: ADD YOUR CODE HERE
+  return reviewers.find((value)=>{
+   return value.books.some(( title)=>{
+    return title.title ===reviewTitle
+    })
+  })
 }
-// console.log(getReviewerByReviewTitle("The Overstory", reviewers));
+console.log(getReviewerByReviewTitle("The Overstory", reviewers));
 
-/**************************************************************
-- Receives a query (string) and an array of reviewer objects,
-  and returns an array of the reviewer objects that contain the query in their name/description. 
-- Hint: uses string method .includes() and iteration method .filter()
- ****************************************************************/
+// /**************************************************************
+// - Receives a query (string) and an array of reviewer objects,
+//   and returns an array of the reviewer objects that contain the query in their name/description. 
+// - Hint: uses string method .includes() and iteration method .filter()
+//  ****************************************************************/
 function searchReviewers(query, reviewers) {
-  //TODO: ADD YOUR CODE HERE
+  return reviewers.filter((reviewers)=>{
+    return reviewers.reviewerName.includes(query)
```

Comment:
Here you're supposed to check if the query has anything to do with the `reviewerName` and `description`, therefore adding an or operator to include the description is the way to go.

Also adding `.toLowercase` method to the strings is the best in this scenario.
```
 return reviewers.filter((reviewers) => {
  let reviewerName = reviewers.reviewerName.toLowerCase();
  let description = reviewers.description.toLowerCase();
  return reviewerName.includes(query) || description.includes(query);
});
```